pipes/WDL/tasks/tasks_16S_amplicon.wdl

```diff
@@ -61,7 +61,7 @@ task qiime_import_from_bam {
     }
     runtime {
         docker: docker
-        memory: "~{memory_mb} MiB"
+        memory: "~{memory_mb} GB"
```

Comment:
Wait no, don't ask for 7000GB of RAM! If you change this to GB, 1) change the variable name from `memory_mb` to `memory_gb` and 2) change the default values down by a factor of 1000.
```suggestion
        memory: "~{memory_mb} MiB"
```